README.md

```diff
@@ -4,6 +4,18 @@ The implementation of the DAO dashboard.
 
 [![ContinuousBuild](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/api3dao/api3-dao-dashboard/actions/workflows/main.yml)
 
+## Running dashboard on Mainnet in docker container
+
+The decentralized approach of being a DAO member is to run API3 dashboard on your local machine. For that you will just need `git` and `docker`:
+```
+git clone --depth=1 git@github.com:api3dao/api3-dao-dashboard.git
+cd api3-dao-dashboard
+docker build -t api3-dao-dashboard .
```

Comment:
Maybe we could push the image on api3 docker hub (https://hub.docker.com/u/api3) to avoid the need to clone the repo beforehand?

Reply:
Yeah that might be nice

Reply:
yes, this is a valuable idea